src/main/java/io/reactivex/rxjava3/core/Observable.java

```diff
@@ -12202,10 +12202,9 @@ public final void safeSubscribe(@NonNull Observer<? super T> observer) {
     }
 
     /**
-     * Returns an {@code Observable} that applies a specified accumulator function to the first item emitted by the current
-     * {@code Observable}, then feeds the result of that function along with the second item emitted by the current
-     * {@code Observable} into the same function, and so on until all items have been emitted by the current {@code Observable},
-     * emitting the result of each of these iterations.
+     * Returns an {@code Observable} that emits the first value emitted by the current {@code Flowable}, then emits one value
```

Comment:
`the current {@code Flowable}`

should be 

`the current {@code Observable}`